src/salary-app/net-to-gross/netToGross.test.js

```diff
@@ -46,12 +46,12 @@ describe('Net to Gross', () =>{
         sinon.restore();
     })
 
-    test(' should be equal 0 when Net is less than or equal 0', async () => {
-
+    test('should return object with gross equal to 0 when Net is less than or equal 0', async () => {
         const inputNet = -19;
         await netToGross(inputNet).then(function (actual) {
-            assert.equal(actual.gross, 0)
-        })
+            expect(actual).to.be.an('object');
```

Comment:
Since you’re learning about Promises, it’s fine to mix `await` with `then` for now.

However, for practicality and team consistency, it’s best to stick to one style.
Just a heads-up!